src/Symfony/Component/Mailer/Event/MessageEvent.php

```diff
@@ -70,10 +71,25 @@ public function isQueued(): bool
         return $this->queued;
     }
 
+    public function isRejected(): bool
+    {
+        return $this->rejected;
+    }
+
+    public function reject(): void
+    {
+        $this->rejected = true;
+    }
+
+    public function accept(): void
```

Comment:
I don't think we need this method.

Reply:
II wasn't sure if you needed this function. Personally, I also think that this function is unnecessary.